vampire_survivors/src/main.py

```
@@ -0,0 +1,343 @@
+import pygame, sys
+from player import Player
+from weapon import Weapon
+from enemy import Enemy
+from projectile import Projectile
```

Ruff (0.12.2)

`projectile.Projectile` imported but unused

Remove unused import: `projectile.Projectile`

(F401)